packages/aws-library/src/aws_library/ec2/client.py

```diff
@@ -131,7 +131,9 @@ async def start_aws_instance(
 
             instances = await self.client.run_instances(
                 ImageId=instance_config.ami_id,
-                MinCount=number_of_instances,
+                MinCount=min(
+                    1, number_of_instances
+                ),  # NOTE: this ensures as many instances as possible are created
```

Comment:
I am not sure I understand this note. 

As far as I understand now the "Count" is set between 
`min(1, number_of_instances)< x < number_of_instances`  but not sure what the implications are.

I wonder what is the different between your option and `1 < x < number_of_instances` provided that possible number_of_instances is probably at least one?

Reply:
adapted